common/src/main/scala/explore/components/ConnectionsStatus.scala

```diff
@@ -54,8 +55,12 @@ object ConnectionsStatus {
         <.span(
           ctx.clients.ExploreDBConnectionStatus(renderStatus("Hasura DB")).when(false),
           ctx.clients.ODBConnectionStatus(renderStatus("ODB")),
-          Button(size = Tiny)(^.onClick --> ctx.clients.odb.close().runAsyncCB)("Close ODB")
-            .when(false)
+          Button(size = Tiny)(
+            ^.onClick --> ctx.clients.odb
+              .disconnect(WSJSCloseParams(code = 4000).asInstanceOf[ctx.clients.odb.CP])
```

Comment:
I'm working of ways to improve the types in `clue` so that this cast is not necessary. I will investigate using `sttp` but it will take a while, so I'm pushing this in the meantime not to hold things up.